src/mongocrypt-marking.c

```diff
@@ -703,6 +761,40 @@ _mongocrypt_fle2_placeholder_to_insert_update_ciphertext (
 }
 
 
+/**
+ * Payload subtype 11: FLE2InsertUpdatePayloadV2
```

Comment:
Suggest updating the comment in mc-fle2-insert-update-payload-private.h to replace ` uint8_t fle_blob_subtype = 4;` with ` uint8_t fle_blob_subtype = 11`.

Reply:
Assuming you mean `mc-fle2-insert-update-payload-private-v2.h` here, and updated that file.